Assignment2/ruliii-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{	
+	"@context":{
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"vocab": "http://www.ontologies.org/ontologies/",
+		"name":"vocab:name",
+		"homeTown":"vocab:homeTown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"description": "http://rdf.data-vocabulary.org/#description",
+		"subject":{
+			"@id": "http://www.ontologies.org/ontologies/#subject",
+			"@type":"xsd:string"
+		}
```

Comment:
Not really sure about what you want to do here. "subject" is an attribute describing the name of each object defined in your array of subjects. Thus, define it as the other properties.